src/config/support-page.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 return [
-    'enquiry_url' => 'https://systems.hiav.networkrail.co.uk/home/forms/enquiry/start',
+    'enquiry_route' => 'https://systems.hiav.networkrail.co.uk/home/forms/enquiry/start',
```

Comment:
The library must work the same between systems.

It is very easy to add the support link to the system during rollout, add it to your deployment plan.